build-system/compile/build.conf.js

```diff
@@ -27,6 +27,7 @@ const defaultPlugins = [
   localPlugin('transform-amp-extension-call'),
   localPlugin('transform-html-template'),
   localPlugin('transform-version-call'),
+  getRewritePlugin(),
```

Comment:
Why are lines 30 and 31 written as wrapper functions? Why not directly call `localPlugin()` here and everywhere else?

Reply:
So that we can minimally transform files under `node_modules`: https://github.com/ampproject/amphtml/pull/26123/files#diff-69b4ced885f765149f7d79017937928bR470